installer/install.iss.in

```diff
@@ -7,8 +7,9 @@
 #else
 #define APP_VERSION   '%APPVERSION%'
 #endif
-#define APP_URL       'http://msysgit.github.io/'
-#define APP_BUILTINS  'etc\fileList-builtins.txt'
+#define MINGW_BITNESS '%MINGW_BITNESS%'

```

Comment:
I added that define so the file can be used with `mingw32` and `mingw64`. The substitution will take place in the `release.sh`